Description: use the system-wide java binary because the source tarball doesn't contain a copy of it
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

Index: chromium-135.0.7049.52/third_party/closure_compiler/compiler.py
===================================================================
--- chromium-135.0.7049.52.orig/third_party/closure_compiler/compiler.py
+++ chromium-135.0.7049.52/third_party/closure_compiler/compiler.py
@@ -13,8 +13,9 @@ import subprocess
 
 
 _CURRENT_DIR = os.path.join(os.path.dirname(__file__))
-_JAVA_PATH = os.path.join(_CURRENT_DIR, "..", "jdk", "current", "bin", "java")
-assert os.path.isfile(_JAVA_PATH), "java only allowed in android builds"
+_JAVA_BIN = "java"
+_JDK_PATH = os.path.join(_CURRENT_DIR, "..", "jdk", "current", "bin", _JAVA_BIN)
+_JAVA_PATH = _JDK_PATH if os.path.isfile(_JDK_PATH) else _JAVA_BIN
 
 class Compiler(object):
   """Runs the Closure compiler on given source files to typecheck them
